dotcom-rendering/src/web/components/YoutubeBlockComponent.importable.tsx

```diff
@@ -88,11 +90,29 @@ export const YoutubeBlockComponent = ({
 	duration,
 	origin,
 	stickyVideos,
+	showOverlayCaption,
 }: Props) => {
 	const [consentState, setConsentState] = useState<ConsentState | undefined>(
 		undefined,
 	);
 
+	useEffect(
+		function hideOverlayCaptionOnPlay() {
+			const hideOverlayCaption = () => {
+				if (showOverlayCaption) {
+					const caption: HTMLElement | null =
+						document.querySelector('.overlay-caption');
+					if (caption) caption.style.display = 'none';
+				}
+			};
+			document.addEventListener('video:play', hideOverlayCaption);
+
+			return () =>
+				document.removeEventListener('video:play', hideOverlayCaption);
+		},
+		[showOverlayCaption],
+	);
```

Comment:
I think this logic needs to be part of the YouTubeBlock component itself. We already have [code there](https://github.com/guardian/atoms-rendering/blob/main/src/YoutubeAtom.tsx#L96) to react to the play event and hide overlays.